freqtrade/exchange/binance.py

```diff
@@ -25,36 +26,55 @@ class Binance(Exchange):
         "l2_limit_range": [5, 10, 20, 50, 100, 500, 1000],
     }
 
-    def stoploss_adjust(self, stop_loss: float, order: Dict) -> bool:
+    _supported_trading_mode_collateral_pairs: List[Tuple[TradingMode, Collateral]] = [
+        # TradingMode.SPOT always supported and not required in this list
+        # (TradingMode.MARGIN, Collateral.CROSS),  # TODO-lev: Uncomment once supported
+        # (TradingMode.FUTURES, Collateral.CROSS),  # TODO-lev: Uncomment once supported
+        # (TradingMode.FUTURES, Collateral.ISOLATED)  # TODO-lev: Uncomment once supported
+    ]
+
+    def stoploss_adjust(self, stop_loss: float, order: Dict, side: str) -> bool:
         """
         Verify stop_loss against stoploss-order value (limit or price)
         Returns True if adjustment is necessary.
+        :param side: "buy" or "sell"
         """
-        return order['type'] == 'stop_loss_limit' and stop_loss > float(order['info']['stopPrice'])
+
+        return order['type'] == 'stop_loss_limit' and (
+            side == "sell" and stop_loss > float(order['info']['stopPrice']) or
+            side == "buy" and stop_loss < float(order['info']['stopPrice'])
```

Comment:
I'd prefer to have parenteses around the 2 conditions (similar to how it's done in the kraken class).

While technically not necessary, having the parenteses simplifies understanding of the intend.